app/auth/authenticate.js

```diff
@@ -22,15 +22,20 @@ export async function getAuth (request, getJwtPublicKeyFunc = getJwtPublicKey) {
     if (!authHeader) {
       return {}
     }
+    logger.verbose('#FCP - Request authentication - Check verification', { code: DAL_REQUEST_AUTHENTICATION_001 })
     const token = authHeader.split(' ')[1]
     const decodedToken = jwt.decode(token, { complete: true })
     const header = decodedToken.header
     const signingKey = await getJwtPublicKeyFunc(header.kid)
     const verified = jwt.verify(token, signingKey)
-    logger.health('#authenticate - JWT verified', { code: FCP_REQUEST_AUTHENTICATION_001 })
+    logger.debug('#FCP Request authentication - JWT verified', { code: DAL_REQUEST_AUTHENTICATION_001 })
```

Comment:
if we are going to be using debug logs we will need to be cautious about what is sent in logs.
Probably should avoid logging any tokens, tbh I quite like the idea of the health level, could be set the same level as info which is probably what we would want to run prod on.

Reply:
I removed health as its not required any more for dashboard.

Was thinking that we would set server level to verbose so we dont get all response data in debug.... but i will have a rethink now i have seen found we have an extra level or two